modules/operator-upgrade.adoc

```diff
@@ -43,6 +43,133 @@ To update {productname} from one minor version to the next, for example, 3.9 ->
 
 For `z` stream upgrades, for example, 3.9.1 -> 3.9.2, updates are released in the major-minor channel that the user initially selected during install. The procedure to perform a `z` stream upgrade depends on the `approvalStrategy` as outlined above. If the approval strategy is set to `Automatic`, the {productname} Operator upgrades automatically to the newest `z` stream. This results in automatic, rolling {productname} updates to newer `z` streams with little to no downtime. Otherwise, the update must be manually approved before installation can begin.
 
+[id="config-editor-removal"]
+== Removing config editor objects on {productname} Operator
+
+The config editor has been removed from the {productname} Operator on {ocp} deployments. As a result, the `quay-config-editor` pod no longer deploys, and users cannot check the status of the config editor route. Additionally, the Config Editor Endpoint no longer generates on the {productname} Operator *Details* page. 
+
+Users with existing {productname} Operators who are upgrading from 3.7, 3.8, or 3.9 to {producty} must manually remove the {productname} config editor by removing the `pod`, `deployment`, `route,` `service`, and `secret` objects.
+
+To remove the `deployment`, `route,` `service`, and `secret` objects, use the following procedure. 
+
+.Prerequisites 
+
+* You have deployed {productname} version 3.7, 3.8, or 3.9.
+* You have a valid `QuayRegistry` object.
+
+.Procedure
+
+. Obtain the `quayregistry-quay-config-editor` route object by entering the following command:
++
+[source,terminal]
+----
+$ oc get route
+----
++
+.Example output
++
+[source,terminal]
+----
+---
+quayregistry-quay-config-editor-c866f64c4-68gtb   1/1     Running     0          49m
+---
+----
+
+. Remove the `quayregistry-quay-config-editor` route object by entering the following command:
++
+[source,terminal]
+----
+$ oc delete route quayregistry-quay-config-editor
+----
+
+. Obtain the `quay-quay-config-editor` deployment object by entering the following command:
```

Comment:
to be consistent, pls change all "quay-quay-xxx" to "quayregistry-quay-xxx" (quite a few in this section).  I worked on 2 env, which has different quay registry name, to be a formal doc, suggest to use only one, "quayregistry-quay-xxx" is more common.